src/GafferML/Inference.cpp

```diff
@@ -60,6 +60,31 @@ using namespace GafferML;
 namespace
 {
 
+std::vector<std::string> customOpLibraryPaths()
+{
+	std::vector<std::string> path_list;
```

Comment:
Please use camelCase to match the prevalent coding style throughout Gaffer.

Reply:
addressed here 6f5be1b85acd1b28627a9ebdd37ee840820071c9